doc/grafana/README.md

```diff
@@ -23,7 +23,7 @@ To quickly test metrics locally (without installing Prometheus Grafana in the cl
     ```
 3. In another terminal, use `kubectl port-forward` to expose the controller's service locally
     ```bash
-    kubectl port-forward service/devworkspace-controller-metrics 8443:8443 &;
+    kubectl port-forward service/devworkspace-controller-metrics 8443:8443 &&
```

Comment:
well, I'm not sure if it's typo.
`command1 && command2`: in my understanding command2 won't be launched until command1 ends, and it's the case when command1 is expected to run until it's interrupted.
But I let @amisevsk provide the truth here )

Reply:
```
[ibuziuk@fedora ~]$  kubectl port-forward service/devworkspace-controller-metrics 8443:8443 &;
bash: syntax error near unexpected token `;'

```

Reply:
@ibuziuk which shell do you use?

it's how it works for me on zsh.
![Screenshot_20210910_164717](https://user-images.githubusercontent.com/5887312/132863399-2257dbd0-411d-40c7-8393-637a6e91d521.png)
![Screenshot_20210910_164705](https://user-images.githubusercontent.com/5887312/132863397-32823591-0863-46bf-b1bb-447b82f29081.png)
Note there is difference ^

Reply:
yeah, hmm 
I'm using vanilla bash 
```
[ibuziuk@fedora ~]$ echo "$SHELL" 
/bin/bash
[ibuziuk@fedora ~]$ ps -p $$
  PID TTY          TIME CMD
25736 pts/2    00:00:00 bash
```

Reply:
I get the feeling this is a quirk in bash that was smoothed over in `zsh` without me noticing. Serhii is correct that we don't want to use `&&` since `kubectl port-forward` does not terminate, so I think a plain `&` is the right option (we want to run the first command in the background and then run the second.

Some background: the semicolon is there from muscle memory (e.g. `sleep 2; sleep 1` to run two commands in a one-liner) -- the port forward command was one line originally and I split it for documentation without removing the `;`. I originally had both port-forward commands together, separated by a semicolon, then realized I needed the first to run in the background so I added the `&`.

In bash and zsh,
```bash
(sleep 2; echo hi) & (sleep 1; echo bye)
```
works as expected, whereas
```bash
(sleep 2; echo hi) &; (sleep 1; echo bye)
```
only works in zsh.

Reply:
```suggestion
    kubectl port-forward service/devworkspace-controller-metrics 8443:8443 &
```